src/controllers/socialAuthController.js

```diff
@@ -131,22 +131,12 @@ class SocialAuthController {
    * @memberof SocialAuthController
    */
   static getToken(request, response) {
-    const { id, username, isNewUser } = request.user;
+    const { id, username } = request.user;
     const payload = { id, username };
     const time = {};
     time.expiresIn = '240h';
     const token = generateToken(payload, time);
-
-    if (isNewUser) {
-      return response.status(201).json({
-        message: 'Signed up successfully',
-        token
-      });
-    }
-    return response.status(200).json({
-      message: `Welcome back ${username}`,
-      token
-    });
+    return response.redirect(`${process.env.APP_BASE_URL}/social?token=${token}`);
```

Comment:
`APP_BASE_URL` is already being used elsewhere in the codebase.

Don't you think this would cause some conflict of some sort when we set that to be the client app's base url?

How about making this one `CLIENT_APP_URL`?

Reply:
@okoroemeka you haven't modified this change.

Reply:
Resolved.

Reply:
👍